services/migrations/0013_change_gdpr_url_to_url_field.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 2.2.10 on 2020-05-13 08:19
```

Comment:
Redoing the migrations so that only one new migrations is added (`0012_add_gdpr_url.py`) would look nice.

Reply:
Yeah. Huge merge in between (also had to merge the migrations). Have to do this manually.